src/richie/apps/courses/templates/courses/cms/_organization_list-item_snippet.html

```diff
@@ -0,0 +1,16 @@
+{% if current_page.publisher_is_draft %}
```

Comment:
This template is redundant with an almost identical one from `fix-check-if-page-extension-is-published` branch on the way to be promoted to a pull request from @sampaccoud. It will still until the new PR has been merged, since tests here depends from this HTML.